odl/space/base_tensors.py

```diff
@@ -496,6 +568,11 @@ def available_dtypes():
         """
         raise NotImplementedError('abstract method')
 
+    @property
+    def array_type(self):
```

Comment:
Is this needed for this PR?

Reply:
Not really, I used it in `DiscreteLp.__getitem__`, but that can be done in a different way. Removed.